tests/testing/asserters_test.py

```diff
@@ -166,6 +166,37 @@ def test_assert_almost_equal(self, size, left_as_arkouda, right_as_arkouda):
                 assert_almost_equal(df, df3, atol=atol, rtol=rtol)
             assert_almost_equivalent(convert_left(df), convert_right(df3), atol=atol, rtol=rtol)
 
+    @pytest.mark.skip_if_max_rank_less_than(2)
+    @pytest.mark.parametrize("size", pytest.prob_size)
+    @pytest.mark.parametrize("left_as_arkouda", [True, False])
+    @pytest.mark.parametrize("right_as_arkouda", [True, False])
+    def test_assert_almost_equal_multi_dim(self, size, left_as_arkouda, right_as_arkouda):
+
+        both_ak = left_as_arkouda and right_as_arkouda
+        convert_left = self.get_converter(left_as_arkouda)
+        convert_right = self.get_converter(right_as_arkouda)
+
+        shape = (2, 2, size)
+
+        rng = ak.random.default_rng()
+        atol = 0.001
+        rtol = 0.001
+        a = ak.arange(size * 4, dtype="float64")
+        a2 = self.perturb(a, atol=atol, rtol=rtol, rng=rng)
+        a3 = a + rtol + atol
+
+        a = a.reshape(shape)
+        a2 = a2.reshape(shape)
+        a3 = a3.reshape(shape)
+
+        if both_ak:
+            assert_almost_equal(a, a2, atol=atol, rtol=rtol)
+        assert_almost_equivalent(convert_left(a), convert_right(a2), atol=atol, rtol=rtol)
+        with pytest.raises(AssertionError):
+            if both_ak:
+                assert_almost_equal(a, a3, atol=atol, rtol=rtol)
+            assert_almost_equivalent(convert_left(a), convert_right(a3), atol=atol, rtol=rtol)
```

Comment:
I was gonna point out that when `both_ak`, I think this will only test `assert_almost_equal` and not `assert_almost_equivalent`. Because we'd exit the `with` block once the first `AssertionError` is raised. They both get called for the `a` and `a2` comparison, so I wanted to make sure this is what we wanted

But looking into it more it seems that `assert_almost_equivalent` just calls `assert_almost_equal` anyway, so I don't this really matters

Reply:
This is a good catch.  I made the change here and will put in a separate PR to fix the rest of the module.